packages/react-core/src/components/DescriptionList/DescriptionListGroup.tsx

```diff
@@ -1,21 +1,35 @@
 import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
 import { css } from '@patternfly/react-styles';
+import { Card, CardProps } from '../Card';
 
 export interface DescriptionListGroupProps extends React.HTMLProps<HTMLDivElement> {
   /** Any elements that can be rendered in the list group */
   children?: any;
   /** Additional classes added to the DescriptionListGroup */
   className?: string;
+  /** Use Card component as a wrapper of DescriptionListGroup */
+  isCard?: boolean;
+  /** Additional props added to the Card component */
+  cardProps?: CardProps;
 }
 
 export const DescriptionListGroup: React.FunctionComponent<DescriptionListGroupProps> = ({
   className,
   children,
+  isCard,
+  cardProps,
   ...props
-}: DescriptionListGroupProps) => (
-  <div className={css(styles.descriptionListGroup, className)} {...props}>
-    {children}
-  </div>
-);
+}: DescriptionListGroupProps) => {
+  if (isCard) {
+    return <Card {...cardProps}>{children}</Card>;
```

Comment:
I think you need to pass `component="div"` to the Card component to fix the a11y violations